R/trace.R

```diff
@@ -61,8 +61,10 @@ quietly <- function(expr) {
 #' @return Nothing; called for its side effects
 #' @export
 stop_mocking <- function() {
-  safe_untrace(curl::form_file)
-  invisible(safe_untrace("request_perform", add_headers))
+  safe_untrace("request_perform", add_headers)
```

Comment:
No reason to treat these two functions differently

```suggestion
  safe_untrace("request_perform", getNamespace("httr"))
```